REVIEWING.md

```diff
@@ -0,0 +1,42 @@
+# Pull Request Reviewing Guidelines for C#/.NET Driver Documentation
```

Comment:
Copy/paste artifact? I assume this is supposed to be "Ruby Driver Documentation".

Reply:
sure is. good catch